package.json

```diff
@@ -45,11 +43,17 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "enzyme": "^3.9.0",
+    "@typescript-eslint/eslint-plugin": "^1.9.0",
+    "@typescript-eslint/parser": "^1.9.0",
     "enzyme-adapter-react-16": "^1.11.2",
+    "enzyme": "^3.9.0",
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-plugin-css-modules": "^2.11.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-react": "^7.13.0",
     "husky": "^1.3.1",
     "lint-staged": "^8.1.4",
-    "prettier": "1.16.4"
+    "prettier": "^1.17.1"
```

Comment:
More of a question than a concern but is there a reason to unpin this?

Reply:
Ummm no I just copy pastaed from a tutorial I think